Packs/SlashNextPhishingIncidentResponseV2/Playbooks/playbook-SlashNext_-_Brand_Phishing_Protection_(HMRC).yml

```diff
@@ -0,0 +1,1084 @@
+contentitemexportablefields:
+  contentitemfields:
+    fromServerVersion: ""
+    itemVersion: ""
+    packID: 8c760e09-e77c-4c64-8baf-db5307d79f58
+    propagationLabels:
+    - all
+    toServerVersion: ""
+description: SlashNext offers Brand Protection against Brand specific phishing attacks.
+  Brands can setup their abuse inbox where user can report specific domains/URLs which
+  are suspected to phish users using that specific brand. Then SlashNext Phishing
+  IR can determine if the reported abuse is credible and using the specific brand
+  for phishing. Playbook can then trigger an email to domain hosting service to take
+  that domain down with evidence.
+id: SlashNext - Brand Phishing Protection (HMRC)
+inputs:
+- description: Text body of the email received at the abuse inbox setup for brand
+    specific phishing protection.
+  key: email_text
+  playbookInputQuery:
+  required: true
+  value:
+    simple: ${incident.labels.Email/text}
+- description: HTML body of the email received at the abuse inbox setup for brand
+    specific phishing protection.
+  key: email_html
+  playbookInputQuery:
+  required: true
+  value:
+    simple: ${incident.labels.Email/html}
+name: SlashNext - Brand Phishing Protection (HMRC)
+outputs: []
+rolename:
```

Comment:
@adi88d How can I modify this commit, its not assigned to me? I actually created the commit through Hackathon instance which is not available now. Also we already have a pack named SlashNextPhishingIncidentResponse, can we make this part of that pack??

Reply:
@SaadatAbid I will change the files... we need to do a quick demo to the playbooks before merge it to our branch, let me know when we can do it next week, i want to see the playbook:
SlashNext - Brand Phishing Protection (HMRC)

and from [this](https://github.com/demisto/content/pull/9161/) pull request:
- SlashNext - Abuse Inbox Management
- SlashNext Abuse Inbox Automation for Email Phishing Protection

thanks

Reply:
@adi88d That's Great. Thank. Please let me know when you want to schedule the demo, we are usually available from 10am UTC to 3pm UTC from Mondays to Fridays.

Reply:
Meanwhile we do have the demo videos for you

[HMRC](https://app.box.com/s/9p7plz6a1oohptbohetfgfao1ftftuzk)
[AbuseInbox](https://app.box.com/s/wudu7l26sdafvq0bm5pcshdyrkxy2ct8)

Reply:
@SaadatAbid Tomorrow at 11am UTC works for you?

Reply:
@adi88d Sure we can have the Demo today but we will need our partner license renewed before that so that we can run the demo for you

Can you provide us the new partner license?

Reply:
@SaadatAbid I will provide you a new partner license and we will schedule a new demo

Reply:
@adi88d Thanks. My email address is saadat.abid.2540@slashnext.net